app/models/app-model.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs/promises')
+
+exports.fetchEndpoint = () => {
+    return fs.readFile('/Users/kamillamohamed/Northcoders/backend/be-nc-news/endpoints.json')
+    .then((data) => {
+        return JSON.parse(data.toString())
+    })
+    .catch((err) => {
+        console.log(err)
+    })
```

Comment:
So this function is requiring in fs to read a json file to a string, parse that string back to a json file and sending that json to the controller and back to the client. Can you think of any way to streamline this?